tests/Feature/CycleMenuNotifyCommandTest.php

```
@@ -128,14 +146,34 @@ public function test_send_cycle_menu_daily_notifications_job_sends_notifications
                 && Str::contains($data['message'] ?? '', 'Today\'s menu');
         });
     }
+
+    public function test_notifications_only_sent_to_tenant_members(): void
+    {
+        // Given a user in a different tenant
+        $otherTenant = Tenant::factory()->create();
+        $otherUser = User::factory()->create(['current_tenant_id' => $otherTenant->id]);
+        $otherTenant->members()->attach($otherUser->id, ['role' => 'owner']);
+
+        $menu = $this->seedActiveMenuForToday();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$menu'. (undefined)

(UnusedLocalVariable)